src/charts/mini-tooltip.js

```diff
@@ -378,6 +378,22 @@ define(function(require){
             return this;
         }
 
+        /**
+         * Gets or Sets the formatter function for the value displayed on the tooltip.
+         * Setting this property makes the tooltip ignore numberFormat. Set by default to
+         * d3-format formatter with numberFormat.
+         * @param  {Function} _x Desired formatter function
+         * @return {Function | module} Current valueFormatter or Chart module to chain calls
+         * @public
+         */
```

Comment:
Could you add an @example tag showing how would you use this accessor?

Reply:
Example added for tooltip and mini-tooltip